Taskfile.dist.yaml

```diff
@@ -1,7 +1,8 @@
 version: "3"
 
 env:
-  DJANGO_SETTINGS_MODULE: "aap_eda.settings.development"
+  EDA_MODE: development
```

Comment:
I dont think we need this. We never had an `EDA_MODE` before, no need to add it now

Reply:
`EDA_MODE: development` is a feature introduced in DAB for selecting `aap_eda.settings.development_defaults` on top of defaults. It needs to be in the list of `environments=("production", "development", "testing"),`